src/networking.c

```diff
@@ -4417,16 +4324,16 @@ size_t getClientMemoryUsage(client *c, size_t *output_buffer_mem_usage) {
      * i.e. unused sds space and internal fragmentation, just the string length. but this is enough to
      * spot problematic clients. */
     mem += c->argv_len_sum + sizeof(robj *) * c->argc;
-    mem += multiStateMemOverhead(c);
+    mem += c->mstate ? multiStateMemOverhead(c) : 0;
```

Comment:
Doesn't `multiStateMemOverhead` return 0 in this case, so we don't need the conditional code here?

```Suggestion
    mem += multiStateMemOverhead(c);
```